src/multi-select/arrow.tsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
+interface IArrowProps {
+  expanded: boolean | undefined;
+}
 
-export const Arrow = ({ expanded }) => (
+export const Arrow = ({ expanded }: IArrowProps) => (
   <svg
     width="24"
     height="24"
-    fill="none"
-    stroke="currentColor"
-    strokeWidth="2"
-    className="dropdown-heading-dropdown-arrow gray"
+    className="rmsc__arrow-svg dropdown-heading-dropdown-arrow gray"
```

Comment:
Instead of using inline styles for the fill and stroke attributes, consider using CSS classes to style the SVG. This would allow you to more easily customize the appearance of the arrow and make the component more reusable.